TestingTravis/TestClass2.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  TestClass2.swift
+//  TestingTravis
+//
+//  Created by Zomato on 17/03/19.
+//  Copyright © 2019 tester. All rights reserved.
+//
+
+import Foundation
+
+class TestClass2{
+    
+    init(){
```

Comment:
<table data-meta="generated_by_1">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Nakul97/TestingTravis/blob/619ac6a6a10c6f7a22a034fd8f271f9af66db0a8/TestingTravis/TestClass2.swift#L13"></span>Opening braces should be preceded by a single space and on the same line as the declaration.<br />
<code>opening_brace</code> <code>TestClass2.swift:13</code></td>
    </tr>
  </tbody>
</table>